### ISSUE TITLE
[feature request] add "Seen By"

### ISSUE THREAD

**Issue by Hongbo-Miao** [yakyak/yakyak#457]
It will be great to add the feature "Seen By".
Thanks for the amazing app!

**Comment by averissimo** [comment]
@Hongbo-Miao partial implementation in 1.4.0-beta3

see #451 and activate option via `view -> experimental`

The library that yakyak uses does not support (yet) having this information when opening a conversation, only after first interaction

**Comment by Hongbo-Miao** [comment]
Thank you for the info!

**Comment by averissimo** [comment]
duplicate of #87 closing.
